### DISCUSSION TITLE
ERROR 20, when trying to move the U-axis, although "Remap ABC to UVW" is set in the Web Builder

### DISCUSSION THREAD

**Discussion by GrblGru** [grblHAL/core#252] [Q&A]
Hi all,

I need the axes XYZ and U for a foam cutter on an Arduino DUE.
For this I have selected 5 axes in the Web Builder and the switch "Remap ABC to UVW".
But I get ERROR 20 when I try to move the U axis. The A axis can still be moved.
What am I doing wrong ?

**Comment by terjeio** [comment]
My bad - the config is not handled correctly when coming from the Web Builder. I'll fix it shortly.

**Reply by terjeio** [reply]
Should be ok now - I've tested with the STM32F756 board on my bench.

**Comment by GrblGru** [comment]
It works now. :)
Many thanks for the quick help
